trade-finance-manager-ui/styles/hide-if-js-not-enabled.scss

```diff
@@ -0,0 +1,3 @@
+.hide-if-js-not-enabled {
+  visibility: hidden;
+}
```

Comment:
I think we should be able to make use of the js-enabled script by using the following:
```
.hide-if-js-not-enabled {
  visibility: hidden;
}

.js-enabled .hide-if-js-not-enabled {
  visibility: visible;
}
```
And then we shouldn't need the script that removes all the instances of this class

Reply:
I updated the styles near the table instead of in this (the `js-hidden` technically already does the same thing as this, but with `display: none` which I wanted to avoid)